tools/update_runnablejobs.py

```diff
@@ -191,59 +260,88 @@ def add_jobs_to_jobpriority(new_data=None, priority=1, timeout=0, set_expired=Fa
                             JobPriorities.expires,
                             JobPriorities.buildsystem).all()
 
-    # Now loop through the reduced jobs and update the database as needed
-    for job in new_data['results']:
-        _buildsystem = job["build_system_type"]
-        platform = job["platform"]
-        testtype = job["testtype"]
-
-        found = False
-        found_id = None
+    # TODO: write test for this
+    # When the table is empty it means that we're starting the system for the first time
+    # and we're going to use different default values
+    map = {}
+    if not len(db_data) == 0:
+        priority, timeout = 1, 0
```

Comment:
I do not like assigning variables to a set of values, especially when the values are not the same (like None, or 0)